templates/tfengine/components/resources/healthcare_datasets/main.tf

```diff
@@ -16,6 +16,7 @@ limitations under the License. */ -}}
 module "{{resourceName . "name"}}" {
   source  = "terraform-google-modules/healthcare/google"
   version = "~> 2.1.0"
+  provider = google-beta
```

Comment:
I don't recall offhand if `module` supports a `provider` field. My feeling is it will need to go in the cloud healthcare's CFT module and directly be added to the corresponding resource.

Reply:
Updated CFT module to use google-beta. Thanks for the pointer!

Reply:
Updated CFT module, thanks!

Reply:
Np. Once that is released, you will need to update the version of the module to use here.